### PR TITLE
Add userInteraction attr for user opened push

### PR DESCRIPTION
**Pull request by xvonabur**
# Summary

In my app I need to run some JS code after user clicked on local notification. I didn't find any way to determine the moment user clicked on it.
This PR adds additional `userInteraction` attribute to userInfo for push messages that was opened by user.

If somebody approve this changes, I'll add an example.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

### CODE REVIEW

**Comment by TaraSinghDanu** [comment]
Not working...

File: ios/RNCPushNotificationIOS.m [hunk 1]
```diff
@@ -125,6 +125,20 @@ @implementation RNCPushNotificationIOS
...
 }
 
+API_AVAILABLE(ios(10.0))
+static NSDictionary *RCTFormatOpenedUNNotification(UNNotification *notification)
+{
+  NSMutableDictionary *formattedNotification = [RCTFormatUNNotification(notification) mutableCopy];
+  UNNotificationContent *content = notification.request.content;
+    
+  NSMutableDictionary *userInfo = [content.userInfo mutableCopy];
+  userInfo[@"userInteraction"] = @"1";
```
Comment (Dallas62) [inline ios/RNCPushNotificationIOS.m:135]: Using a number would make this compliant with the remote userInteraction value.
```obj-c
userInfo[@"userInteraction"] = [NSNumber numberWithInt:1];
```

**Review by Dallas62 — CHANGES_REQUESTED** [review]
Thanks for this PR, this is an awesome fix for the community, I have only one change to request from my tests 😉

**Comment by nishanttatva** [comment]
@Dallas62 @xvonabur 
ETA when this will be merged?

**Comment by Dallas62** [comment]
Hi @nishanttatva 
We are not able to merge, so the answer is no 😉

File: ios/RNCPushNotificationIOS.m (hunk 1 above)
Comment (xvonabur) [reply ios/RNCPushNotificationIOS.m]: Thanks! Good catch.

**Comment by xvonabur** [comment]
@Dallas62 I've added requested changes. Feel free to review it again.

**Review by Dallas62 — APPROVED** [review]
LGTM

**Comment by Dallas62** [comment]
@Naturalclar It would be great if you find time to look at this PR 😉

**Review by Dallas62 — APPROVED** [review]

**Comment by Naturalclar** [comment]
@Dallas62 thanks for the ping! I'll take a look at it soon

**Review by Naturalclar — APPROVED** [review]
@xvonabur Thank you! The change looks good to me.
Would you be able to add an example usage in the README and example?

**Comment by xvonabur** [comment]
@Naturalclar I've added example and information to README. Can you check it?

**Comment by Naturalclar** [comment]
@xvonabur thank you for the change! I've published `1.5.0` with the new feature :tada:

**Pull request merged**